src/screen.rs

```diff
@@ -26,11 +26,21 @@ impl Screen {
     pub fn new() -> Result<Screen> {
         let stdout = MouseTerminal::from(AlternateScreen::from(stdout().into_raw_mode()?));
         Ok(Screen {
-               size: termion::terminal_size()?,
+               size: termion::terminal_size().chain_err(|| ErrorKind::TerminalSize)?,
                stdout: stdout,
            })
     }
 
+    /// Update the terminal size and return `true` if the height changed, and false otherwise.
+    pub fn resize(&mut self) -> Result<Option<(u16, u16)>> {
+        let new_size = termion::terminal_size()?;
+        if self.size.1 == new_size.1 {
```

Comment:
Sure sure. So how about we start with an impl of `Eq` that only considers the height of the tuple? Then when we're ready to handle line lengths, we expand the impl to include the width